From 1cc044fb3b00c290afcacab3ca044d371bb039be Mon Sep 17 00:00:00 2001
From: Maxim Uvarov <maxim.uvarov@linaro.org>
Date: Wed, 9 Apr 2014 18:42:29 +0400
Subject: [PATCH] add dependancy for -lrt

Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>
---
 snort-2.9.6.0/src/snort.c |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/snort-2.9.6.0/src/snort.c b/snort-2.9.6.0/src/snort.c
index 1481d4f..0280107 100644
--- a/snort-2.9.6.0/src/snort.c
+++ b/snort-2.9.6.0/src/snort.c
@@ -777,6 +777,12 @@ static int InlineFailOpen (void)
  */
 int main(int argc, char *argv[])
 {
+	/* hack to add deps to -lrt for timer_create() and friends from odp */
+	static volatile int link_rt = 0;
+	timer_t timerid = 0;
+	if (link_rt)
+		timer_settime(timerid, 0, NULL, NULL);
+
 #if defined(WIN32) && defined(ENABLE_WIN32_SERVICE)
     /* Do some sanity checking, because some people seem to forget to
      * put spaces between their parameters
-- 
1.7.9.5

