From 71aeb1a6bbda6464b717565aa4a1e3126cb59f1c Mon Sep 17 00:00:00 2001
From: Maxim Uvarov <maxim.uvarov@linaro.org>
Date: Wed, 16 Apr 2014 18:16:57 +0400
Subject: [PATCH 2/2] remove static from needed functions

Signed-off-by: Maxim Uvarov <maxim.uvarov@linaro.org>
---
 src/snort.c |   14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/snort.c b/src/snort.c
index ce5c52e..5e5060a 100644
--- a/src/snort.c
+++ b/src/snort.c
@@ -485,7 +485,7 @@ static void InitProtoNames(void);
 static const char* GetPacketSource(char**);
 
 static void CleanExit(int);
-static void SnortInit(int, char **);
+void SnortInit(int, char **);
 static void InitPidChrootAndPrivs(pid_t);
 static void ParseCmdLine(int, char **);
 static int ShowUsage(char *);
@@ -515,8 +515,8 @@ static void FreeReferences(ReferenceSystemNode *);
 static void FreePlugins(SnortConfig *);
 static void FreePreprocessors(SnortConfig *);
 
-static void SnortUnprivilegedInit(void);
-static int SetPktProcessor(void);
+void SnortUnprivilegedInit(void);
+int SetPktProcessor(void);
 static void PacketLoop(void);
 #if 0
 static char * ConfigFileSearch(void);
@@ -1618,7 +1618,7 @@ static Packet s_packet;
 static DAQ_PktHdr_t s_pkth;
 static uint8_t s_data[65536];
 
-static DAQ_Verdict PacketCallback(
+/*static*/ DAQ_Verdict PacketCallback(
     void* user, const DAQ_PktHdr_t* pkthdr, const uint8_t* pkt)
 {
     int inject = 0;
@@ -1779,7 +1779,7 @@ static DAQ_Verdict PacketCallback(
     return verdict;
 }
 
-static void PrintPacket(Packet *p)
+void PrintPacket(Packet *p)
 {
     if (p->iph != NULL)
     {
@@ -2989,7 +2989,7 @@ static void ParseCmdLine(int argc, char **argv)
  */
 // TBD add GetDecoder(dlt) to decode module and hide all
 // protocol decoder functions.
-static int SetPktProcessor(void)
+int SetPktProcessor(void)
 {
     const char* slink = NULL;
     const char* extra = NULL;
@@ -5162,7 +5162,7 @@ static DAQ_Verdict IgnoreCallback (
 // packet passing is done by the driver/hardware.  the goal then is to put as
 // much initialization stuff in SnortInit() as possible and to restrict this
 // function to those things that depend on DAQ startup or non-root user/group.
-static void SnortUnprivilegedInit(void)
+void SnortUnprivilegedInit(void)
 {
 #ifdef ACTIVE_RESPONSE
     // this depends on instantiated daq capabilities
-- 
1.7.9.5

